Doc/library/codecs.rst

```diff
@@ -1417,8 +1420,8 @@ to :class:`bytes` mappings. They are not supported by :meth:`bytes.decode`
 |                      | quotedprintable, | quoted printable.            | ``quotetabs=True`` /         |
 |                      | quoted_printable |                              | :meth:`quopri.decode`        |
 +----------------------+------------------+------------------------------+------------------------------+
-| uu_codec             | uu               | Convert the operand using    | :meth:`uu.encode` /          |
-|                      |                  | uuencode.                    | :meth:`uu.decode`            |
+| uu_codec             | uu               | Convert the operand using    | :meth:`!uu.encode` /         |
```

Comment:
We can mention that they are deprecated (if there's enough space in the table)

Reply:
Something like https://github.com/python/cpython/pull/111070/commits/828687752c8953727e0c9e5995a80a97b25d85c8?

![image](https://github.com/python/cpython/assets/1324225/93d9eff7-02cc-4665-bccc-69cf786ecfe1)


Or as a `[2]` footnote?

Reply:
1. seems better, if the whole table won't need a refactoring
2. is also fine 👍

Reply:
I'll update the 3.11 backport to match this one.